### ISSUE TITLE
"Disk or ISO image (Please choose)" Incorrect translation of the Polish string.

### ISSUE THREAD

**Issue by ghost** [pbatard/rufus#1916]
﻿

Checklist
---------
- [ ] I looked at https://github.com/pbatard/rufus/wiki/FAQ to see if my question has already been answered.
- [ ] I performed a search in the issue tracker for similar issues using keywords relevant to my problem, such as the error message I got from the log.
- [ ] I clicked the 'Log' button or pressed <kbd>Ctrl</kbd>-<kbd>L</kbd> in Rufus, and copy/pasted the log into the line that says `<FULL LOG>` below.
- [ ] The log I am copying is the FULL log, starting with the line `Rufus version: x.y.z` - I have NOT removed any part of it.

Additionally (if applicable):
- [ ] I ran a bad blocks check, by clicking _Show advanced format options_ then _Check device for bad blocks_, and confirmed that my USB is not defective.
- [ ] I also tried one or more of the following:
  - [ ] Using a different USB drive.
  - [ ] Plugging the USB into a different port.
  - [ ] Running Rufus on a different computer.
- [ ] If using an image, I clicked on the `(✓)` button to compute the MD5, SHA1 and SHA256 checksums, which are therefore present in the log I copied. I confirmed, by performing an internet search, that these values match the ones from the official image.

Issue description
-----------------
Unable to select a DVD drive as the source for rufus or it's an incorrect translation.

Missleading translation.

"Dysk lub obraz ISO" is equal in meaning to "Select DVD/BluRay/CD Disc or Diskette or Hard Drive or Partition or ISO image";

"Obraz dysku lub ISO" == "The image of the drive or ISO"

In Polish the word "Image"("Obraz") needs to come before the thing that is being enumerated, if it is after the first enumeration then it counts as the thing added to the individual Nth item rather than all N items.

Log
---
```
<FULL LOG>
CRITICAL ERROR: Human read a piece of string in his native tongue, he is going to make the wrong assumption, better watch out.
```

**Comment by pbatard** [comment]
Obviously I don't speak Polish, and I also can't blame translators for making mistakes, so I will ask you to please tell me exactly what you'd like to see knowing that what you are able to select is a DVD image or a BluRay image or a CD image or a Diskette image or a Hard Drive image or an ISO image (which is really the same as the first 3), and not a physical optical disc media inserted into a physical DVD drive.

Basically, what you want to do (select a physical optical media that is inserted into a physical DVD drive) will not work, because it is not a feature that Rufus provides.

So, knowing this, what would you suggest the translation should be?

**Comment by temporaryFuckMicrosoft** [comment]
Any of the following:

Wybierz obraz dysku lub płyty (Choose an image of a disk[raw image] or disc[iso])
Obraz płyty lub dysku (proszę wybrać) (Image of a disc or disk (please choose))

**Comment by temporaryFuckMicrosoft** [comment]
#1917
